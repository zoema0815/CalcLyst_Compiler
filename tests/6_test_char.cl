int main () {
    char c;
    c = 'c';
    printChar(c);
    c = '1';
    printChar(c);
    c = 'F';
    printChar(c);
    c = '\n';
    printChar('a');
    return 0;
}