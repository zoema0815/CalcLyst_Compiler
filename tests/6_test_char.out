c
1
F
a
